### PR TITLE
Feature: Title was improved, file added in utils for its generation

### PR DESCRIPTION
**Pull request by arturolinares24**
Changes made @rotorrest

### CODE REVIEW

File: templates/sales_order_performance/paths/sales_order_perfomance.py [hunk 1]
```diff
@@ -80,7 +84,9 @@ def plot_kpi_indicators(self):
             },
         ]
 
-        self.shimoku.plt.indicator(data=indicator_data, order=self.order)
+        self.shimoku.plt.indicator(
+            data=indicator_data, order=self.order, color="success"
+        )
```
Comment (rotorrest) [inline templates/sales_order_performance/paths/sales_order_perfomance.py:89]: I was not expecting this in the PR, why is this added to the PR?

File: templates/sales_order_performance/paths/sales_order_perfomance.py [hunk 2]
```diff
@@ -98,6 +104,7 @@ def plot_charts(self):
             order=self.order,
             x="Month",
             rows_size=30,
+            padding="2",
```
Comment (rotorrest) [inline templates/sales_order_performance/paths/sales_order_perfomance.py:107]: I was not expecting this in the PR, why is this added to the PR? x2

File: templates/sales_order_performance/paths/sales_order_perfomance.py [hunk 3]
```diff
@@ -107,6 +114,7 @@ def plot_charts(self):
...
```
Comment (rotorrest) [inline templates/sales_order_performance/paths/sales_order_perfomance.py:117]: I was not expecting this in the PR, why is this added to the PR? x3

**Review by rotorrest — COMMENTED** [review]
I left questions, please double-check if everything is ok, merge =)

File: templates/sales_order_performance/paths/sales_order_perfomance.py (hunk 1 above)
Comment (arturolinares24) [reply templates/sales_order_performance/paths/sales_order_perfomance.py:89]: I adjusted it so that the graphics would look a bit better.

File: templates/sales_order_performance/paths/sales_order_perfomance.py (hunk 2 above)
Comment (arturolinares24) [reply templates/sales_order_performance/paths/sales_order_perfomance.py:107]: I adjusted it so that the graphics would look a bit better.

File: templates/sales_order_performance/paths/sales_order_perfomance.py (hunk 3 above)
Comment (arturolinares24) [reply templates/sales_order_performance/paths/sales_order_perfomance.py:117]: I adjusted it so that the graphics would look a bit better.
